includes/class-wc-stripe-checkout-session-lifecycle.php

```
@@ -0,0 +1,244 @@
+		}
+
+		try {
+			$data = $this->manager->synchronize();
+		} catch ( Exception $e ) {
+			WC_Stripe_Logger::error( 'Native classic checkout session synchronization failed.', [ 'error_message' => $e->getMessage() ] );
+			$data = $this->manager->mark_failed( $e->getMessage() );
+		}
+
+		$fragments[ self::CLASSIC_FRAGMENT_SELECTOR ] = $this->get_classic_fragment_html( $data );
+		return $fragments;
+	}
```

PHPMD (2.15.0)

[error] The method add_classic_fragment is not named in camelCase. (undefined)

(CamelCaseMethodName)